### ISSUE TITLE
Error fetching Alfresco data

### ISSUE THREAD

**Issue by marcogianini** [fcorti/Alflytics#23]
My environment is (only server):
Cent OS 7.6.1810
Alfresco 5.2
Postgresql 9.6.14

In Postgresql it has the Alfresco database (alfresco), the Alflytics database (alflytics) and the postgres database.

I made the installation by connecting to the Postgresql database according to the manual.

Through the Spoon on my desktop I made the connection in Pentaho and changed the 5 scripts it has in Alflytics \ ETL successfully, pointing to postgresql and the Pentaho host url. In PeNtaho I created an ODBC Alflytics pointing to the Alflytics database in Postgresql.

When running Job Get all on Spoon, it does not feed the Warehouse tables, only Operational and Staging. And with that it returns no data to generate report.

And does pentaho.log and catalina.log show no direction for this error?

help me please?

[spoon.log](https://github.com/fcorti/Alflytics/files/3378332/spoon.log)
![Screen Extract Alflytics](https://user-images.githubusercontent.com/40039143/60983772-f236e980-a310-11e9-8673-7892020482df.JPG)
[pentaho.log](https://github.com/fcorti/Alflytics/files/3378339/pentaho.log)

[catalina.log](https://github.com/fcorti/Alflytics/files/3378345/catalina.log)

**Comment by fcorti** [comment]
Hi Marco,

The error I see is the following one:

pt.webdetails.cpf.packager.dependencies.JsMinifiedDependency] Error getting input stream for dependency saikuWidget/SaikuWidgetComponent.js.

Does it happen during the ETL extraction or during the dashboard usage?

**Comment by marcogianini** [comment]
Corti, good morning,

This error seems to me to be of the Widget Saiku, but at that moment I am not using it. And this error may be the lack of installation of Cent OS dependencies, I'm verifying.

My only concern is that when executing the extract it does not input into the Data WareHouse tables. This does not generate the reports because it has no data.

I did a new installation on my Windows machine with Postgresql 9.5, Alfresco 5.2 and Pentaho 7.1.0.0-12 and also did not input the data warehouse tables.

We are already using Postgresql 9.5 because of the incompatibility of SQL Query commands (INSERT .. ON..).

Do you have any special parameters in creating the Alflytics Database?

You need some specific logging to help me. I can send you from my machine that I made the installation yesterday.

Thanks

**Comment by marcogianini** [comment]
Corti, good morning,

This error seems to me to be of the Widget Saiku, but at that moment I 
am not using it. And this error may be the lack of installation of Cent 
OS dependencies, I'm verifying.

My only concern is that when executing the extract it does not input 
into the Data WareHouse tables. This does not generate the reports 
because it has no data.

I did a new installation on my Windows machine with Postgresql 9.5, 
Alfresco 5.2 and Pentaho 7.1.0.0-12 and also did not input the data 
warehouse tables.

We are already using Postgresql 9.5 because of the incompatibility of 
SQL Query commands.

Do you have any special parameters in creating the Alflytics Database?

You need some specific logging to help me. I can send you from my 
machine that I made the installation yesterday.

Thanks

Em 12/07/2019 05:31, Francesco Corti escreveu:

**Comment by fcorti** [comment]
Hi Marco,

I don't think the Saiku error is related to the issue you are experiencing.
Saiku is about the visualization of data, the ETL is another task, using different technologies.

Looking at your dashboard, it seems you are not using audit.

On the repository extraction: the Alfresco to Staging goes well with 256 pieces of content, the extraction from Staging to Operational goes well with all the pieces of content, but the Operational to Warehouse fails. 

I tried to check in the log file, but the only error I can see is in about Saiku (and it is not relted to the ETL).

I suggest you to launch the ETL on Repository only (with a clean catalina.log file)  and check into the log, once completed. You should find the reason of the problem.

Let me know how it goes.

**Comment by marcogianini** [comment]
Hi,
I ran it again today, however I was not allowed to run on the client.
So I executed on my machine presenting the same scenario. On my machine it does not have the Saiku Analytics Plugin.
My ambiance is this way:
Windows 10
Postgresql 9.5
Pentaho Server CE 7.1.0.0-12
Pentaho Data Integrator CE 7.1.0.0-12
Alflytics v5.0.EA
Attached are the files catalina.log and pentaho.log
[pentaho.2019-07-15.log.log](https://github.com/fcorti/Alflytics/files/3392821/pentaho.2019-07-15.log.log)
[catalina.2019-07-15.log.log](https://github.com/fcorti/Alflytics/files/3392822/catalina.2019-07-15.log.log)
